docs/concepts/extra-attributes.md

```diff
@@ -52,6 +52,24 @@ const paragraphExtension = new ParagraphExtension({
 
 This example accomplishes the same things as the previous example and remirror is smart enough to automatically parse the dom and write to the dom the required values.
 
+### Render as data attributes
+
+You can return an array of a key value pair, to determine how your extra attribute is rendered in the DOM
+
+```ts
+import { ParagraphExtension } from 'remirror/extension/paragraph';
+
+const paragraphExtension = new ParagraphExtension({
+  extraAttributes: {
+    custom: {
+      default: 'my default',
+      parseDOM: (dom) => dom.getAttribute('data-custom'),
+      toDOM: (attrs) => (['data-custom', attrs.custom]),
```

Comment:
```suggestion
      toDOM: (node) => (['data-custom', node.attrs.custom]),
```

Reply:
I'm not sure this is correct, I think it is `attrs` here. Due to how the function is called [here](https://github.com/remirror/remirror/blob/3a89f121cb10b5ef6a0b2705aa5b9e65a4ad469e/packages/%40remirror/core/src/builtins/schema-extension.ts#L877)?

Reply:
Again, you're absolutely correct :woman\_facepalming: